tools/flaky-test-reporter/result.go

```diff
@@ -158,6 +158,29 @@ func addSuiteToRepoData(suite *junit.TestSuite, buildID int, rd *RepoData) {
 	}
 }
 
+// https://github.com/knative/test-infra/issues/2120
+func filterOutParentTests(originalCases []junit.TestCase) []junit.TestCase {
+	parents := make(map[string]interface{})
```

Comment:
There is a nicer `set` to use :)
https://godoc.org/k8s.io/apimachinery/pkg/util/sets